cds-plugin.js

```diff
@@ -52,6 +55,16 @@ const rmFiles = async (dir, exts) => fs.existsSync(dir)
     )
     : undefined
 
+cds.on('on-watch', async () => {
```

Comment:
The event name sounds redundant.
```suggestion
cds.on('watch', async () => {
```